<html class="has-navbar-fixed-top" style="scroll-behavior: smooth;">
  <head>
    <meta charSet="utf-8"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>
    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon.png"/>
    <title>
      nfdi4plants - News
    </title>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
    <link rel="stylesheet" type="text/css" href="/style/style.css"/>
    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@creativebulma/bulma-collapsible@1.0.4/dist/css/bulma-collapsible.min.css"/>
    <script defer="true" src="https://kit.fontawesome.com/0d3e0ea7a6.js" crossOrigin="anonymous"></script>
    <script type="module" src="/js/bundle.js"></script>
    <style>
      
                    body {
                        margin: 0px;
                    }

                    thead tr th, strong {
                        color: var(--accent-text-color) !important
                    }

                    a {
                        color: var(--link-color, #4FB3D9) !important;
                    }

                    a:hover {
                        color: var(--link-hover-color, #3A3A3A) !important;
                    }

                    thead {
                        font-size: 1.2rem;
                    }

                    nfdi-toc, nfdi-body {
                        --outside-background-color: rgb(240, 243, 246);
                        --header-color: rgb(10, 12, 16)
                        --element-text-color: #0E1116;
                        --element-background-color: #fff;
                        --accent-text-color: rgb(31, 194, 167);
                        --link-color: #4FB3D9;
                        --link-hover-color: #8ad3ee;
                    }
                
    </style>
  </head>
  <nfdi-navbar></nfdi-navbar>
  <body>
    <section class="hero is-medium is-dark has-bg-lightblue">
      <div class="hero-body">
        <div class="container has-text-centered">
          <h1 class="title">
            
          </h1>
        </div>
      </div>
    </section>
    <section class="section">
      <div class="container p-4">
        <h1 class="title is-1">
          MAdLand presents at DOMPS Plant Science Seminar
        </h1>
        <h2 class="subtitle is-2">
          23 Feb 2024
        </h2>
        <div class="content">
          <p><img src="../../images/News-Items/madland-logo.png" :="" style="float:right" alt="The MAdLand logo with the text “Molecular Adaptation to Land: plant evolution to change”" /></p>
<p>Deepti Varshney and Saskia Hiltemann from the University of Freiburg presented a talk entitled &quot;MAdLand Resources &amp; Tools&quot; at the Plant Science Seminar organised by the <a href="http://www.plantphotobiology.org/DOMPS/Plant_Science_Seminar.php">Department of Molecular Plant Sciences (DOMPS)</a>, Institute of Biology, University of Freiburg on February 23, 2024.</p>
<p>Both are involved in the <a href="https://madland.science/">MAdLand</a> (<strong>M</strong>olecular <strong>Ad</strong>aptation to <strong>Land</strong>) project, a DFG-funded research consortium exploring the molecular mechanism behind the transition from water to land, from alga to land plant. MAdLand is committed to support and promote FAIRification of all research data generated within its projects, through the use of comprehensive research data management (RDM) strategies for the effective acquisition, processing, archival, and sharing of research data sets. This goal strongly aligns with DataPLANT aims, and therefore MAdLand heavily relies on DataPLANT resources and tools (e.g. <a href="https://github.com/nfdi4plants/ARC-specification">ARCs</a>) to achieve this goal, and is committed to a close collaboration with DataPLANT, in line with the previously signed <a href="https://nfdi4plants.org/content/news/2022-07-08-madland-and-dataplant-signed-a-memorandum-of-understanding.html"><em>Memorandum of Understanding</em></a> between MAdLand and NFDI4Plants on joint collaboration in research data management in fundamental plant research.</p>
<p>As a further part of this collaboration, MAdLand is committed to making its collection of plant research tools and resources openly and freely available to the scientific community, with the support of <a href="https://www.denbi.de">de.NBI</a>.</p>
<p>All work presented in the seminar presentation is part of this collaboration between <a href="https://madland.science">MAdLand</a>, <a href="https://nfdi4plants.org">DataPLANT (NFDI4Plants)</a>, <a href="https://www.denbi.de">de.NBI</a> &amp; <a href="https://galaxyproject.org">Galaxy</a> and covered the core set of resources useful for plant researchers: MAdLandDB (GenomeZoo), Galaxy, TAPscan v4, and <a href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">DataPLANT ARCs</a>.</p>
<p><a href="https://galaxyproject.org">Galaxy</a> is a widely used open-source platform that enables user-friendly bioinformatics while adhering to the FAIR (findable, accessible, interoperable, reusable) science principles. The <a href="https://training.galaxyproject.org/">Galaxy Training Network (GTN)</a> employs a collaborative, open, and FAIR approach to scientific training materials. With over 300 tutorials authored and reviewed by a global community, the GTN serves researchers, educators, and scientific tool developers, this is a valuable resource for any researcher looking to perform their own data analysis.</p>
<p><strong>MAdLandDB</strong>, also known as the <strong>Genome Zoo</strong>, offers a comprehensive protein sequence database accessible via Galaxy. The database includes protein sequences of various lineages, including, e.g., fungi, animals, phylo-diverse algae, bacteria, and archaea, with the focus on non-seed plants and streptophyte algae. It provides reliable protein sequences for comparative sequence analyses.</p>
<p><a href="http://tapscan.plantcode.cup.uni-freiburg.de">TAPscan v4</a> (currently in beta) is an advanced tool for genome-wide annotation of plant transcription-associated proteins (TAPs). TAPs (TFs and TRs) are key to understanding the development and evolution of plant form and function. Access to reliable, up-to-date classifications of TAPs enables comparative analyses that expand our knowledge of plant transcriptional regulation.</p>
<p><a href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">ARCs (Annotated Research Contexts)</a> represent an important part of a plant researcher's RDM (research data management) strategy. ARCs are developed by DataPLANT (NFDI4Plants) and are an RO-crate implementation optimised for plant research data. DataPLANT has developed useful tooling around ARCs, that will aid plant researchers in their creation and publication. The MAdLand project is committed to creating ARCs for each of its funded projects. Several MAdLand ARCs have already been <a href="https://archive.nfdi4plants.org/search?q=&amp;l=list&amp;p=1&amp;s=10&amp;sort=newest">published</a>, and more are set to be released in the coming months.</p>
<p><img src="../../images/News-Items/madland-domps-seminar-picture.png" :="" width="45%" alt="Picture of Saskia Hiltemann presenting the MAdLand tools and resources at the DOMPS plant science seminar. Several attendees can be seen in the foreground, the current slide behind her is entitled “Why ARCs?” and shows a visual representation of the data management lifecycle of research projects." /></p>
<p><strong>Acknowledgements:</strong> The speakers would like to thank the organisers of DOMPS for providing the opportunity to present a talk at the seminar. The speakers would also like to thank the MAdLand/Rensing lab, Galaxy, DataPLANT and de.NBI.</p>
<p><strong>Links</strong></p>
<ul>
<li><a href="https://tinyurl.com/madland-resources">Link to presentation slides</a></li>
<li><a href="http://www.plantphotobiology.org/DOMPS/Plant_Science_Seminar.php">Link to seminar series</a></li>
</ul>

        </div>
      </div>
    </section>
  </body>
  <nfdi-footer></nfdi-footer>
</html>